gpii/node_modules/transformer/test/html/TransformerTests.html

```diff
@@ -15,6 +15,12 @@
         <script src="../../../../../node_modules/infusion/src/framework/core/js/ModelTransformation.js"></script>
         <script src="../../../../../node_modules/infusion/src/framework/core/js/ModelTransformationTransforms.js"></script>
 
+        <!-- These are required not for the tests themselves, but for the coverage client -->
+        <script type="text/javascript" src="../../../../../node_modules/infusion/src/framework/core/js/FluidDocument.js"></script>
```

Comment:
Probably consider to remove `type="text/javascript"` since: 1. unnecessary; 2. be consistent with the use of `<script>` tags at other spots. This comment applies across other files in the pull request.